scripts/driver/driver_internal.sh

```diff
@@ -0,0 +1,174 @@
+#!/bin/bash
```

Comment:
I'm assuming this file is mostly copied from the old one (no diff shown here). Please manually diff them to double check one last time as you've probably been working on this for a while ;)

Reply:
I wish github detected renames.

https://gist.githubusercontent.com/trironkk/feb822f407ac3d534806aef128c81b97/raw/cf0ea9ddae6157c9b342fed55eb32f6828308cc0/gistfile1.txt